x-pack/docs/en/security/authorization/document-level-security.asciidoc

```diff
@@ -3,14 +3,24 @@
 === Document level security
 
 Document level security restricts the documents that users have read access to.
-In particular, it restricts which documents can be accessed from document-based 
-read APIs. 
+In particular, it restricts which documents can be accessed from document-based
+read APIs.
 
 To enable document level security, you use a query to specify the documents that
-each role can access. The document query is associated with a particular data
+each role can access. The document `query` is associated with a particular data
 stream, index, or wildcard (`*`) pattern and operates in conjunction with the
 privileges specified for the data streams and indices.
 
+The specified `query`:
+
+* Expects the same format as if it was defined in the search request
```

Comment:
This is mostly true except it can also use a templated query. However this feature is not documented on this page, but on the other page of ["Setting up field and document level security"](https://elasticsearch_64426.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/field-and-document-access-control.html#templating-role-query). It is cross-linked at the end of this page.

Reply:
👍 I'll add a separate bullet with a link to the templating section of that page.